### ISSUE TITLE
How to render dynamic routes?

### ISSUE THREAD

**Issue by vitorhugosg** [chrisvfritz/prerender-spa-plugin#412]
Hello, how are you?

I have a marketplace project, I would like to know if I can render the dynamic routes of the products, would anyone know if I can do this?

Thanks in advance.

- [ ]

**Comment by valentinoli** [comment]
Yes, you can. But you need to know the routes in advance since the `routes` options only accepts an Array. It really depends on where you store the route information. Are the product ids stored in a database? How do you fetch them?

**Comment by trdbau** [comment]
@valentinoli can you show me way to make a dynamic route for item in marketplace?
